examples/network-policies/README.md

```diff
@@ -0,0 +1,3 @@
+# Network Policies
+
+Demonstrates how to use Kubernetes Network Policies with EKS.
```

Comment:
This is very laconic.

Reply:
Added more details and a mermaid diagram